addons/website_sale/views/product_views.xml

```diff
@@ -184,7 +184,7 @@
                     />
                 </group>
             </xpath>
-            <xpath expr="//page[@name='sales']//group[@name='description']" position="after">
+            <xpath expr="//page[@name='sales']//group[@name='description']" position="before">
```

Comment:
Why do you need this change?

Reply:
So the ecommerce setting can be in cluster together as in ecommerce shop -> ecommerce media -> ecommerce description.